packages/RdfService/README.md

```diff
@@ -19,3 +19,6 @@ const rdfService = new RdfService("http://localhost:3030/", "ontology");
 const diagrams = await rdfService.runQuery(query);
```

Comment:
What is the 'query' that is being run here?  Is it one to get diagrams from the 'ontology' rdfService?  Or is it doing something else (in which case the destination variable should be named to something other than 'diagrams').